packages/sdk/test/evm/gasless.test.ts

```diff
@@ -47,4 +48,43 @@ describe("Gasless Forwarder", async () => {
 
     await pack.open(0, 0);
   });
+
+  it.skip("should mint batch via multicall tx with relayer", async () => {
+    const NFT_COLLECTION_ADDRESS = "0x24A5aB2878B63716B001aa9AbE816c2662192B12";
+    const provider = ethers.getDefaultProvider(RPC_URL);
+    // const wallet = Wallet.createRandom().connect(provider);
+    const wallet = new Wallet("<PRIVATE KEY>", provider);
+    const sdk = new ThirdwebSDK(wallet, {
+      gasless: {
+        openzeppelin: {
+          relayerUrl: "<RELAYER URL>",
```

Comment:
now this I don't think will be feasible in unit tests without lots of work... would it be possible to instead intercept the transaction and see if the packed solidity is in there as expected correctly?

Reply:
Yeah, couldn't test locally. Used a real URL, removed after testing.